### PR TITLE
Homepage edits

### PR DESCRIPTION
**Pull request by ronakJain21**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,12 +1,13 @@
 import './App.css';
 import { BrowserRouter, Routes, Route } from "react-router-dom";
 import Signup from './components/Signup/Signup';
-
+import Homepage from './component/Homepage/Homepage';
```
Comment (aisaaxs) [inline client/src/App.js:4]: Please correct the Homepage path -> './components/Homepage/Homepage';

File: client/src/components/Homepage/Homepage.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
+/* Homepage.css */
```
Comment (aisaaxs) [inline client/src/components/Homepage/Homepage.css:1]: Please use the color palettte from the app icon to portray consistency in UI/UX elements.

The colors to be used are:
--oxford-blue: #01133Dff;
    --blue-green: rgb(11, 166, 209);
    --oxford-blue-2: #02113Aff;
    --oxford-blue-3: #030F36ff;
    --indigo-dye: #154A7Fff;
    --berkeley-blue: #072D53ff;
    --oxford-blue-4: #00204Bff;
    --oxford-blue-5: #01133Aff;
    --oxford-blue-6: #001943ff;
    --oxford-blue-7: #030C31ff;

**Review by aisaaxs — CHANGES_REQUESTED** [review]

File: client/src/App.js (hunk 1 above)
Comment (ronakJain21) [reply client/src/App.js]: thank you for the correction

**Comment by aisaaxs** [comment]
Can you please pull the changes from the development branch and create the Revised-Homepage branch based off of the dev branch? This way we can make sure that no conflicts arise whatsoever. Otherwise, the Homepage functions well and looks pretty good too. If possible, please redesign to match the colour palette of the app logo which is specified in the Signup and Login css files.

**Pull request closed without merging**